_health-care/_js/components/financial-assessment/AnnualIncomeSection.jsx

```diff
@@ -156,6 +205,23 @@ class AnnualIncomeSection extends React.Component {
               value={this.props.data.childrenOtherIncome}
               onValueChange={(update) => {this.props.onStateChange('childrenOtherIncome', update);}}/>
         </div>
+      </div>);
+    }
+
+    if (this.props.reviewSection) {
```

Comment:
well I was using `reviewSection` because `editButton` actually toggles `sectionComplete` between true and false in an effort to give the user the ability to "update" or "save" after they have edited their information. If we use `sectionComplete` for `editButton`, `editButton` would disappear when you click it and you would be stuck looking at input fields without feeling like you have saved the information before hitting the final submit.